Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM praekeltfoundation/django-bootstrap:py3.6-stretch
+
+WORKDIR /casepro
+
+RUN apt-get-install.sh git wget curl && curl -sL https://deb.nodesource.com/setup_4.x | bash && apt-get install nodejs -y
+
+RUN echo "Downloading Casepro from https://github.com/rapidpro/casepro/archive/v1.3.8.tar.gz" && \
+    wget -O casepro.tar.gz "https://github.com/rapidpro/casepro/archive/v1.3.8.tar.gz" && \
```

Comment:
This looks good. But what about having the version number as an env var? That will allow us to do things like https://github.com/praekeltfoundation/courier-docker/blob/master/.travis.yml#L16-L19 , which allows us to check once a day for a new release, and if there is one, automatically building it.